### DISCUSSION TITLE
"Query fired: did not receive all answers in time for graph.facebook.com:443"

### DISCUSSION THREAD

**Discussion by VasilisPH** [oddbit/flutter_facebook_app_events#299] [Q&A]
Hello flutterers!

While everything works as expected  in Android , in iOS it does not. 
I followed widget's instructions to the letter and
I added https://pub.dev/packages/app_tracking_transparency to get the ios 14+ permission dialog about tracking transparency.

Yet this "Query fired: did not receive all answers in time for graph.facebook.com:443" is showing in the Debug Console. 

Any ideas on what could possibly go wrong ?

**Comment by michelandres** [comment]
Hi, Could you resolve this? we have the same problem. Thanks

**Comment by Artein** [comment]
Same for me. Any news on this one?
